backend/server.js

```diff
@@ -20,6 +20,18 @@ app.get("/", (req, res) => {
   res.send("Hello, world!");
 });
 
+app.get("/:userID", (req, res) => {
+  const userID = req.params.userID;
+  const userData = JSON.parse(fs.readFileSync(userDB));
+  const user = userData.find((user) => user.userId === userID);
+  const { userId, userEmail, userPassword } = user;
+  if (!user) {
+    res.status(404).send("User not found");
+  } else {
```

Comment:
This part should come before line 27, this might actually throw an error if the user doesnt exist

Reply:
Also, can you please a try-catch block just in case there are un-foreseen errors they can all be handled appropriately

Reply:
when a user is not found:
![Screenshot 2023-03-01 at 3 12 16 PM](https://user-images.githubusercontent.com/98188684/222266065-edd31b9c-3f99-4e09-a86d-1c4e509cfd5e.png)
when a user is found:
![Screenshot 2023-03-01 at 3 12 31 PM](https://user-images.githubusercontent.com/98188684/222266104-cdb65e88-ff41-48ac-80d5-59a12bc4c36e.png)

Reply:
![Screenshot 2023-03-01 at 3 14 13 PM](https://user-images.githubusercontent.com/98188684/222266390-d7974683-563f-4c6f-8e01-d1976e109751.png)